master/internal/resourcemanagers/job.go

```diff
@@ -51,7 +50,26 @@ func jobStats(taskList *taskList) *jobv1.QueueStats {
 		}
 		reqs = append(reqs, req)
 	}
+	stats := requestsToQueueStats(reqs)
+	return stats
+}
+
+//nolint:deadcode // Method used by Slurm support in determined-ee
+func jobStatsByPool(taskList *taskList, resourcePool string) *jobv1.QueueStats {
```

Comment:
maybe better (more flexible) is just `jobStatsByPool` + `taskList.byPool(name string)` but this is definitely fine

Reply:
Don't understand your comment.   You mean more like this?

//nolint:deadcode // Method jobStatsByPool is used by Slurm support in determined-ee

Reply:
oh I just meant, we wouldn't need `jobStatsByPool` if `taskList.byPool(name string)` existed. sorry, total non-comment. was just thinking `taskList` could probably use the enhancement